ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/434__data-science-pipelines-ui.robot

```diff
@@ -66,6 +67,39 @@ Verify User Can Create, Run and Delete A DS Pipeline From DS Project Details Pag
     Delete Pipeline Server    ${PRJ_TITLE}
     Delete Data Science Project    ${PRJ_TITLE}
 
+Verify No Metadata Pods Were Deployed
+    [Documentation]    Because the product includes code that is currently deactivated, it is imperative to prevent any
+    ...         occurrences of disabled elements from appearing unexpectedly.
+    [Tags]    Smoke
+    ...       Tier1
+    Create Pipeline Server    dc_name=${DC_NAME}
+    ...    project_title=${PRJ_TITLE}
+    Wait Until Pipeline Server Is Deployed    project_title=${PRJ_TITLE}
+    Import Pipeline    name=${PIPELINE_TEST_NAME}
+    ...    description=${PIPELINE_TEST_DESC}
+    ...    project_title=${PRJ_TITLE}
+    ...    filepath=${PIPELINE_TEST_FILEPATH}
+    Pipeline Should Be Listed    pipeline_name=${PIPELINE_TEST_NAME}
+    ...    pipeline_description=${PIPELINE_TEST_DESC}
+    Capture Page Screenshot
+    ${workflow_name}=    Create Pipeline Run    name=${PIPELINE_TEST_RUN_BASENAME}
+    ...    pipeline_name=${PIPELINE_TEST_NAME}    from_actions_menu=${FALSE}    run_type=Immediate
+    ...    press_cancel=${TRUE}
+    Open Data Science Project Details Page    ${PRJ_TITLE}
+    ${workflow_name}=    Create Pipeline Run    name=${PIPELINE_TEST_RUN_BASENAME}
+    ...    pipeline_name=${PIPELINE_TEST_NAME}    from_actions_menu=${FALSE}    run_type=Immediate
+    Open Data Science Project Details Page    ${PRJ_TITLE}
+    Wait Until Pipeline Last Run Is Started    pipeline_name=${PIPELINE_TEST_NAME}
+    ...    timeout=10s
+    Wait Until Pipeline Last Run Is Finished    pipeline_name=${PIPELINE_TEST_NAME}
+    ...    timeout=180s
+    @{pods} =    Oc Get    kind=Pod    namespace=${PRJ_TITLE}
```

Comment:
Lines 96-100 could be simplified using the existing keywords in ods-ci/ods_ci/tests/Resources/Page/OCPDashboard/Pods/Pods.robot like _Check If Pod Exists_